### PR TITLE
[Fire Mage] More Fixes for Heating Up

### PR DESCRIPTION
**Pull request by Sharrq**
Just noticed the Phoenix Flames Suggestion had the Fire Blast Icon.

### CODE REVIEW

File: src/Parser/Mage/Fire/Modules/Features/HeatingUp.js [hunk 1]
```diff
@@ -119,19 +119,21 @@ class HeatingUp extends Analyzer {
   }
 
   suggestions(when) {
-		when(this.fireBlastUtilSuggestionThresholds)
+		when(this.fireBlastMissedPercent).isLessThan(this.fireBlastUtilSuggestionThresholds.isLessThan.minor)
```
Comment (kfinch) [inline src/Parser/Mage/Fire/Modules/Features/HeatingUp.js]: It looks like you undid the suggestion threshold updates by accident...

File: src/Parser/Mage/Fire/Modules/Features/HeatingUp.js [hunk 2]
```diff
@@ -119,19 +119,21 @@ class HeatingUp extends Analyzer {
...
 					.icon(SPELLS.FIRE_BLAST.icon)
-					.actual(`${formatPercentage(this.fireBlastUtil)}% Utilization`)
-					.recommended(`<${formatPercentage(recommended)}% is recommended`);
+					.actual(`${formatPercentage(this.fireBlastMissedPercent)}% missed`)
+					.recommended(`<${formatPercentage(recommended)}% is recommended`)
+					.regular(this.fireBlastUtilSuggestionThresholds.isLessThan.average).major(this.fireBlastUtilSuggestionThresholds.isLessThan.major);
 			});
-      when(this.phoenixFlamesUtilSuggestionThresholds)
-  			.addSuggestion((suggest, actual, recommended) => {
+    when(this.phoenixFlamesMissedPercent).isLessThan(this.phoenixFlamesUtilSuggestionThresholds.isLessThan.minor)
```
Comment (kfinch) [inline src/Parser/Mage/Fire/Modules/Features/HeatingUp.js]: It looks like you undid the suggestion threshold updates by accident...

File: src/Parser/Mage/Fire/Modules/Features/HeatingUp.js (hunk 1 above)
Comment (Sharrq) [reply src/Parser/Mage/Fire/Modules/Features/HeatingUp.js]: oh ... oops .... wonder how that happened

**Comment by Sharrq** [comment]
redid the suggestions.

**Pull request merged**